sdk/src/test/java/io/dapr/client/DaprHttpTest.java

```diff
@@ -74,7 +94,7 @@ public void invokeGetMethod() throws IOException {
                 .get("http://localhost:3500/v1.0/get")
                 .respond(EXPECTED_RESULT);
 
-        DaprHttp daprHttp = new DaprHttp(3500, okHttpClient);
+        DaprHttp daprHttp = new DaprHttp(3500,okHttpClient);
```

Comment:
This thing will be resolved in our checkstyle. Usually, I see checkstyles failing without this space. It is OK now, but we will do a big reformatting in the end of the deliverable.

Reply:
Sorry i missed that detail. i am going to check and fix if there is any other styling problem.

Reply:
We will do a big restyling change once code is stable.